ssc/cmod_utilityrate5.cpp

```diff
@@ -1023,15 +1110,14 @@ class cm_utilityrate5 : public compute_module
 						&revenue_w_sys[0], &payment[0], &income[0],
 						&demand_charge_w_sys[0], &energy_charge_w_sys[0],
 						&monthly_fixed_charges[0], &monthly_minimum_charges[0],
-						&monthly_dc_fixed[0], &monthly_dc_tou[0],
 						&monthly_ec_charges[0],
 						&monthly_ec_charges_gross[0],
 						&monthly_excess_dollars_earned[0],
 						&monthly_excess_dollars_applied[0],
 						&monthly_excess_kwhs_earned[0],
 						&monthly_excess_kwhs_applied[0],
-						&dc_hourly_peak[0], &monthly_cumulative_excess_energy[0], &monthly_cumulative_excess_dollars[0],
-						&monthly_bill[0], rate_scale[i], i + 1,
+						&rate.dc_hourly_peak[0], &monthly_cumulative_excess_energy[0], &monthly_cumulative_excess_dollars[0],
```

Comment:
So, all the outputs are zero based now? How do they appear in the Data Tables in SAM?

Reply:
The main change here was pointing the relevant inputs to the rate_data object. The table results are unchanged:

![image](https://user-images.githubusercontent.com/5530592/95889532-bbcff180-0d3f-11eb-8992-03d580d80fb3.png)

Reply:
Ah, yes! That was necessary to line up with some of the inputs like rate_scale and matches the convention in the PV and battery compute modules. The year adjustments for the financial models occurs on line 1364, and the output in those data tables also matches what is expected:

![image](https://user-images.githubusercontent.com/5530592/96475055-cd5a4300-11f0-11eb-8a95-ffb5bd8ffe2a.png)

Does that need additional comments?

Reply:
I think if the results are matching up as before then it's alright as is